spec/spec_helper.rb

```diff
@@ -35,7 +35,7 @@
   config.before(:each) do
     PgAuditLog::Entry.uninstall rescue nil
     connection.tables.each do |table|
-      connection.drop_table_without_auditing(table)
```

Comment:
This method doesn't exist in this repo any more. It wouldn't have been exercised unless a spec failed and its table didn't get cleaned up. We need to use `execute` instead of `drop_table` because `drop_table` includes audit trigger code that will break in this scenario.